pytorch_lightning/loggers/base.py

```diff
@@ -143,10 +153,12 @@ def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None):
         pass
 
     @staticmethod
-    def _convert_params(params: Union[Dict[str, Any], Namespace]) -> Dict[str, Any]:
-        # in case converting from namespace
+    def _convert_params(params: Union[Dict[str, Any], Namespace, DictConfig]) -> Dict[str, Any]:
+        # in case converting from Namespace or hydra's DictConfig
         if isinstance(params, Namespace):
             params = vars(params)
+        elif USING_HYDRA_CONF:
+            params = OmegaConf.to_container(params, resolve=True)
```

Comment:
> You are assuming that if Hydra is installed the user is always passing an OmegaConf object config object. this may not always be the case.

This should do it. 
```suggestion
    def _convert_params(params: Union[Dict[str, Any], Namespace, DictConfig]) -> Dict[str, Any]:
        # in case converting from Namespace or hydra's DictConfig
        if isinstance(params, Namespace):
            params = vars(params)
        elif USING_HYDRA_CONF and isinstance(params, DictConfig):
            params = OmegaConf.to_container(params, resolve=True)
```

For point 2 and 3, it's require more discussion and decision to integrate with Lightning, so currently outside of the scope for this PR.

If the fix for point 1 above is OK, I think this for its limited scope, this PR is completed, cc @Borda.

Reply:
cc: @PyTorchLightning/core-contributors 

it seems that this or is only about converting param to the Hydra format and what is the application then?